_infra/helm/sample/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 12.0.6
+version: 12.0.7
 
```

Comment:
I wonder if we should bump the version to 12.1.0 as we're removing so much stuff